packages/polaris-common/src/context/paginated-context.ts

```diff
@@ -0,0 +1,4 @@
+export interface PaginatedContext {
```

Comment:
im not sure if this class is necessary, for online paging you anyway dont use the totalCount field, so just add pageSize to the online paging context and add pageSize + totalCount to snapshot paging context.